### PR TITLE
Test coverage

### PR DESCRIPTION
**Pull request by pfdietz**
Fixed some issue from previous pull request that were revealed in CI.

Add tests for extended-sequences on sbcl.
Add tests for coverage (as revealed by sb-cover).

### CODE REVIEW

**Comment by sionescu** [comment]
Please squash these into one commit.

File: tests.lisp [hunk 1]
```diff
@@ -5,6 +5,46 @@
 
 (in-package :split-sequence/tests)
 
+#+sbcl
+(progn
+  (eval-when (:compile-toplevel :load-toplevel :execute)
```
Comment (sionescu) [inline tests.lisp:10]: Is the EVAL-WHEN necessary ?

File: tests.lisp [hunk 2]
```diff
@@ -5,6 +5,46 @@
...
+    (defclass eseq (standard-object sequence)
+      ((actual-seq :type list :initarg :actual-seq :initform nil
+                   :accessor actual-seq))
+      (:documentation "Extended sequence type in SBCL")))
+  (defmethod sb-sequence:length ((s eseq)) (length (actual-seq s)))
```
Comment (sionescu) [inline tests.lisp:15]: Please put the method body on a separate line.

**Review by sionescu — CHANGES_REQUESTED** [review]

File: tests.lisp (hunk 1 above)
Comment (pfdietz) [reply tests.lisp:10]: It was necessary.  I think this is a bug in SBCL, but the class is not available in the following method definitions without that, and warnings are issued.

**Comment by pfdietz** [comment]
Closing this and issuing a pull request from a branch with a squashed merge

**Pull request closed without merging**